rust_src/src/lisp.rs

```diff
@@ -948,6 +948,12 @@ impl LispObject {
         }
     }
 
+    #[inline]
+    pub fn xcons(self) -> LispCons {
+        debug_assert!(self.is_cons());
+        LispCons(self)
+    }
```

Comment:
I think this is worth a method, but I can also remove it.

Reply:
So this is `as_cons_unchecked`?